README.md

```diff
@@ -41,6 +41,13 @@ Instructions for how to get started with developing on the Pantheon codebase. Pl
 * [Code Coverage](docs/development/code-coverage.md)
 * [Logging](docs/development/logging.md)
 
+## Issues 
+
+Pantheon issues are tracked in [Jira](https://pegasys1.atlassian.net/secure/Dashboard.jspa) not GitHub. 
```

Comment:
This will always load their last looked at dashboard. Do we want to specifically send them to the system dashboard (https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10000)?